src/pudl/transform/ferc1.py

```diff
@@ -1541,7 +1541,7 @@ class Ferc1TableTransformParams(TableTransformParams):
     merge_xbrl_metadata: MergeXbrlMetadata = MergeXbrlMetadata()
     align_row_numbers_dbf: AlignRowNumbersDbf = AlignRowNumbersDbf()
     drop_duplicate_rows_dbf: DropDuplicateRowsDbf = DropDuplicateRowsDbf()
-    assign_quarterly_filed_data_to_annual_dbf: AssignQuarterlyDataToYearlyDbf = (
+    assign_quarterly_data_to_yearly_dbf: AssignQuarterlyDataToYearlyDbf = (
```

Comment:
D'oh! I fucked it up. I was editing in the GitHub online editor because I was running a full ETL in the background. Sorry!